usagov/vote-gov-drupal
# VOTE-1123: defer custom js files

## Jira ticket

[VOTE-1123](https://cm-jira.usa.gov/browse/VOTE-1123)

## Description

Deferred uswds.min.js and the 4 custom js files and we have for improved performance (lighthouse testing).

## Deployment and testing

### Post-deploy steps

1. run 'lando drush cr'.

### QA/Testing instructions

1. load up the local site, go to inspect-element
2. verify that the script tags with "external-links" and "theme-switcher" both have the "defer" attribute within their tag

## Checklist for the Developer




## Checklist for the Peer Reviewers
